pkg/gds/config/config_test.go

```diff
@@ -21,6 +21,7 @@ var testEnv = map[string]string{
 	"GDS_ADMIN_ENABLED":              "true",
 	"GDS_ADMIN_BIND_ADDR":            ":444",
 	"GDS_ADMIN_MODE":                 "debug",
+	"GDS_ADMIN_TOKEN_KEYS":           "1y9fT85qWaIvAAORW7DKxtpz9FB:testdata/key1.pem,1y9fVjaUlsVdFFDUWlvRq2PLkw3:testdata/key2.pem",
```

Comment:
This is an example of what an environment variable that is parsed to a map in envconfig should look like; basically a comma or semicolon separated list of key:value pairs.

Reply:
Ah, this is making me think that I was wrong in thinking we were appending to a single `ksuid.pem` file, and that we should be uniquely naming each pem so that we can instead we give `envconfig` a list of the two most recent pem files we created.  I suppose that means we have to remember the name of the 3-months ago pem file (so that both it and the new one will be included). Do we need to worry about deleting the >3 months ago pems?